README.md

```diff
@@ -1,2 +1,217 @@
-# Microverse_hello
-It is about a project which introduces some skills about how to use linters in html and css 
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
+  <h3><b>Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Microverse\_Hello ](#-microverse_hello-)
+	- [🛠 Built With ](#-built-with-)
+		- [Tech Stack ](#tech-stack-)
+		- [Key Features ](#key-features-)
+	- [🚀 Live Demo ](#-live-demo-)
+	- [💻 Getting Started ](#-getting-started-)
+		- [Prerequisites](#prerequisites)
+		- [Setup](#setup)
+		- [Install](#install)
+		- [Usage](#usage)
+		- [Run tests](#run-tests)
+		- [Deployment](#deployment)
+	- [👥 Authors ](#-authors-)
+	- [🔭 Future Features ](#-future-features-)
+	- [🤝 Contributing ](#-contributing-)
+	- [⭐️ Show your support ](#️-show-your-support-)
+	- [🙏 Acknowledgments ](#-acknowledgments-)
+	- [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+	- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse_Hello <a name="about-project"></a>
+
+**Microverse_Hello** is a project about using linter in a project contains html and css
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">html.js</a></li>
+	<li><a href="https://css.org/">css.js</a></li>
+	<li><a href="https://linter.org/">hint</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
```

Comment:
[OPTIONAL] kindly remove all default text under each section of this template after customizing with your own content this will make your work clean and more professional